components/accept-financial-contributions/ApplyToHost.js

```diff
@@ -66,63 +63,116 @@ class ApplyToHost extends React.Component {
+        defaultMessage: 'Become a fiscal host',
+      },
+      interestedInHosting: {
+        id: 'fiscalHost.interestedInHosting',
+        defaultMessage: 'Are you interested in fiscally hosting Collectives?',
+      },
+      seeMoreHosts: {
+        id: 'fiscalHost.seeMoreHosts',
+        defaultMessage: 'See more hosts',
+      },
     });
   }
 
+  setTags = tag => {
+    tag === 'All fiscal hosts' ? this.setState({ tags: null }) : this.setState({ tags: [tag] });
+  };
+
   render() {
-    const { hosts, intl, onChange } = this.props;
+    const { intl, onChange, collective } = this.props;
+    const { tags } = this.state;
 
     return (
       <Fragment>
+        <CollectiveNavbar collective={collective} onlyInfos={true} />
         <Box mb={4} mt={5}>
           <H1 fontSize={['H5', 'H3']} lineHeight={['H5', 'H3']} fontWeight="bold" color="black.900" textAlign="center">
             {intl.formatMessage(this.messages.header)}
           </H1>
         </Box>
-        <Container alignItems="flex-end" display="flex" ml={7} pl={6} width={1}>
-          <img src={umbrellaIllustration} width="264px" height="184px" />
-          <Box maxWidth={'480px'}>
-            <H2 fontSize="H5" color="black.900">
-              {intl.formatMessage(this.messages.applyToHost)}
-            </H2>
-            <P mb={3} lineHeight="LeadParagraph" color="black.700">
-              {intl.formatMessage(this.messages.infoParagraph)}
-            </P>
-          </Box>
-        </Container>
-        <Container alignItems="flex-start" display="flex" px={6} py={[20, 60]} width={1}>
-          <AllCardsContainer>
-            {hosts.map(host => (
-              <CollectiveCardContainer key={`${host.legacyId}-container`}>
-                <HostCollectiveCard
-                  key={host.legacyId}
-                  host={host}
-                  collective={this.props.collective}
-                  onChange={onChange}
-                />
-              </CollectiveCardContainer>
+        <Container px={[1, 7]}>
+          <Container
+            display="flex"
+            alignItems={['center', 'flex-end']}
+            flexDirection={['column', 'row']}
+            width={1}
+            px={[3, 5]}
+            my={3}
+          >
+            <img src={umbrellaIllustration} width="160px" height="160px" />
+            <Box maxWidth={'480px'} ml={[0, 4]}>
+              <H2 fontSize="H5" color="black.900" textAlign={['center', 'left']} my={[3, 0]}>
+                {intl.formatMessage(this.messages.applyToHost)}
+              </H2>
+              <P mb={2} lineHeight="LeadParagraph" color="black.700">
+                {intl.formatMessage(this.messages.infoParagraph)}
+              </P>
+            </Box>
+          </Container>
+          <Flex px={[3, 5]} my={4} flexWrap="wrap">
+            {tagList.map(tag => (
+              <FilterTag key={`${tag}-tag`} px={2} mr={2} onClick={() => this.setTags(tag)}>
+                {tag}
```

Comment:
**[:hammer_and_wrench: **Important**]** We have a component to internationalize collective tags: `I18nCollectiveTags`. It won't work for `All fiscal hosts`, but it would be great to have this filter internationalized too.

---

<details>
<summary>More details about this feedback</summary>
<br/>

This feedback categorization is inspired by [Netlify's Feedback Ladders system](https://www.netlify.com/blog/2020/03/05/feedback-ladders-how-we-encode-code-reviews-at-netlify). It includes 5 levels of feedback:

:rotating_light: **Critical blocker**
This feedback blocks all related work, and requires immediate action. 

:no_entry_sign: **Blocker**
This feedback should block the work from moving forward or being approved, but it does not necessarily require immediate action from the team.

:hammer_and_wrench: **Important**
This feedback should not block the work from moving forward or being approved, but it does require future action from the team. This generally means the feedback is not required for the MVP or initial draft of this work, but should be addressed at some point and we should ideally create an issue to track it.

:sparkles: **Enhancement**
This feedback should not block the work from moving forward or being approved. The project lead or owner should consider the feedback and implement or open an issue it if they agree with it.

:ice_cream: **Details & personal preferences**
This feedback should not block the work from moving forward or being approved. The project lead or owner should consider the feedback and implement or open an issue it if they deem it useful. This is also the category for everything that relates to personal preferences or code style.

</details>